README.md

```
@@ -29,76 +29,90 @@ These schemas enable CloudAI to be flexible and compatible with different system
 ### uninstall
-To uninstall test prerequisites, run CloudAI CLI in uninstall mode:
+The opposite to the install mode, this mode removes installed test prerequisites.
+
 ```bash
 cloudai uninstall\
     --system-config conf/common/system/example_slurm_cluster.toml\
     --tests-dir conf/common/test\
     --test-scenario conf/common/test_scenario/sleep.toml
 ```
+
+### list
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)